internal/pkg/term/selector/selector.go

```diff
@@ -133,6 +160,55 @@ func (s *DeployedService) String() string {
 	return fmt.Sprintf("%s (%s)", s.Svc, s.Env)
 }
 
+// VPC has the user select an available VPC.
+func (s *Ec2Select) VPC(prompt, help string) (string, error) {
+	vpcIDs, err := s.ec2Svc.ListVPC()
+	if err != nil {
+		return "", fmt.Errorf("list VPC ID: %w", err)
+	}
+	if len(vpcIDs) == 0 {
+		return "", ErrVpcNotFound
+	}
+	vpcID, err := s.prompt.SelectOne(
+		prompt, help,
+		vpcIDs)
+	if err != nil {
+		return "", fmt.Errorf("select VPC: %w", err)
+	}
+	return vpcID, nil
+}
+
+// PublicSubnet has the user select public subnets given the VPC ID.
+func (s *Ec2Select) PublicSubnet(prompt, help, vpcID string) ([]string, error) {
+	return s.subnet(prompt, help, vpcID, true)
+}
+
+// PrivateSubnet has the user select private subnets given the VPC ID.
+func (s *Ec2Select) PrivateSubnet(prompt, help, vpcID string) ([]string, error) {
+	return s.subnet(prompt, help, vpcID, false)
+}
+
+func (s *Ec2Select) subnet(prompt, help string, vpcID string, public bool) ([]string, error) {
+	resp, err := s.ec2Svc.ListVPCSubnets(vpcID)
```

Comment:
Usually a boolean parameter is an indicator of a code smell: https://martinfowler.com/bliki/FlagArgument.html

In this situation, I think the right thing to do is something like:
```go
resp, err := s.ec2.ListVPCSubnets(vpcID, ec2.FilterForPublicSubnets)
```
Where `ListVPCSubnets` returns all subnets by default, and the functional options: `FilterForPublicSubnets` or `FilterForPrivateSubnets` do the necessary filtering.